analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/ObjectsShouldNotBeDisposedMoreThanOnceTest.cs

```diff
@@ -71,6 +71,12 @@ public class ObjectsShouldNotBeDisposedMoreThanOnceTest
                 .AddReferences(TestHelper.ProjectTypeReference(projectType))
                 .Verify();
 
+        [TestMethod]
+        public void ObjectsShouldNotBeDisposedMoreThanOnce_Roslyn_VB14() =>
+            roslynVB.AddPaths("ObjectsShouldNotBeDisposedMoreThanOnce.vb")
```

Comment:
Something is missing around here :D 
```suggestion
            roslynVB.AddPaths("ObjectsShouldNotBeDisposedMoreThanOnce.VB14.vb")
```